charters/DataOperations/charter.md

```diff
@@ -0,0 +1,92 @@
+# [Data Operations](mailto:data-operations-team@data.humancellatlas.org)
+
+
+## Description
+
+The Data Coordination Platform (DCP) Data Operations Team is responsible for providing the scientific community with effective tools and resources which maximize the value and use of HCA data. This involves identifying the value added for the HCA consumers from various DCP components and the needs and expectations of the consumers, and connecting these communities through useful, fully vetted products and their distribution.
+
+## Definitions
+
+Data contribution: a set of raw data derived from the same experimental process, and the metadata describing that process
+Data release: the point at which metadata and data from a single data contribution become publicly available
+Data distribution: the packaging of metadata and data from multiple data contributions in meaningful ways and any announcements or documentation supporting it
+
+## Objectives
+
+The objective of the Data Operations is to provide the scientific community with effective tools and resources from the DCP which maximize the value and use of HCA data
+
+## In-scope
+
+- Data operations
+  - Engage with all DCP components to identify deliverables, including data, pipelines, metadata, standards, and user features, which maximize the value and consistency of HCA data
+  - Coordinate DCP-wide end-to-end testing pre-release and continued QC checks post-release to ensure data and data resource integrity and useability through component updates
+  - Identify data release- and data distribution-associated engineering needs of all DCP components and participate in crafting technical specifications
+  - Establish and integrate into the DCP roadmap a reproducible plan that includes the specifications for and the timing of iterative data distribution to the community
+  - Monitor the status of data contributions at each step in the DCP data workflow from initial identification of suitability through data release and potential inclusion in additional data distribution
+  - Engage with Governance Group and Analysis Work Group to identify data quality standards which dictate data eligible to be released and data eligible to be included in data distributions
```

Comment:
Does the Governance group still exist?

Reply:
Any data uploaded to the HCA is publicly available/released, so would the step to determine data eligibility occur before it is put into the HCA?

Reply:
I believe this is squarely in the purview of the Analysis Working Group but feel free to ask around (you are welcome to one or any of our AWG meetings if that is helpful).

I would imagine that what is ment here is "to identify data assurance standards which dictate data eligible to be released and quality control standards for each data distribution". Would this wording fit with what you are planning?

Reply:
Two votes for AWG only is enough for me. Updated in 9c5c574.

@TimothyTickle updated the wording in b881027. With respect to AWG, we specifically are referring to QC